### PR TITLE
push to ghcr.io

### PR DESCRIPTION
**Pull request by bohrasd**
https://theplanttokyo.atlassian.net/browse/SRE-3575

### CODE REVIEW

File: build.sh
Comment (idealhack) [inline build.sh:1]: Is this script only used for testing?

Comment (bohrasd) [reply build.sh:1]: @idealhack The name says "Personal Access Token" so I thought it would be fine to use my token.
Who should generate it btw? :sweat:

Comment (bohrasd) [reply build.sh:1]: Screenshot 2023-10-17 at 4 19 42 PM
It needs a PAT (classic) with these permission here

Comment (idealhack) [reply build.sh:1]: We should probably use @theplant-ci's token, please review https://theplanttokyo.atlassian.net/browse/SRE-851

Comment (idealhack) [reply build.sh:1]: Where is this script being called?

Comment (bohrasd) [reply build.sh:1]: @idealhack Can you generate a new token for @theplant-ci and save to secret `ghcr-token` in `test-pods` namespace in prow cluster? Then I'll update the username here

Comment (bohrasd) [reply build.sh:1]: I've generated the token in theplant-bot account

Comment (idealhack) [reply build.sh:1]: Thanks

**Review by idealhack — APPROVED** [review]

**Pull request merged**